sdk/tables/Azure.Data.Tables/src/Sas/SasExtensions.cs

```diff
@@ -135,6 +192,72 @@ internal static void AppendProperties(this TableSasQueryParameters parameters, S
             }
         }
 
+        /// <summary>
+        /// Builds the query parameter string for the SasQueryParameters instance.
+        /// </summary>
+        /// <param name="parameters"></param>
+        /// <param name="stringBuilder">
+        /// StringBuilder instance to add the query params to
+        /// </param>
+        internal static void AppendProperties(this TableAccountSasQueryParameters parameters, StringBuilder stringBuilder)
```

Comment:
similar to the comment in `TableAccountSasQueryParameters`, we could possibly consolidate the common appended properties here if this extends `SasQueryParameters` instead.